File: week-02/day-4/Workshop 13.4.2023/src/ListIntroduction2.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collection;

public class ListIntroduction2 {
    public static void main(String[] args) {
        ArrayList<String> fruitsA = new ArrayList<>(Arrays.asList("Apple", "Avocado", "Blueberries", "Durian", "Lychee"));
        ArrayList<String> fruitsB = new ArrayList<>(fruitsA);
        //Add "Passion Fruit" and "Pomelo" to List B in a single statement
        //ArrayList<String> fruitsToBeAdded = new ArrayList<>(Arrays.asList("Passion Fruit", "Pomelo"));
        //fruitsB.addAll(fruitsToBeAdded);

        System.out.println(fruitsA.contains("Durian"));
        fruitsB.remove("Durian");
        fruitsA.add(4, "Kiwifruit");

        System.out.println(fruitsA.equals(fruitsB));

        System.out.println(fruitsA.indexOf("Avocado"));
        System.out.println(fruitsB.indexOf("Durian"));

        fruitsB.addAll(new ArrayList<>(Arrays.asList("Passion Fruit", "Pomelo")));

        System.out.println(fruitsA.get(2));
        System.out.println(fruitsA);
        System.out.println(fruitsB);



    }
}

File: week-07/day-2/Workshop 16.5.2023/list_introduction2.py
list_A = ['Apple', 'Avocado', 'Blueberries', 'Durian', 'Lychee']
list_B = ['Apple', 'Avocado', 'Blueberries', 'Durian', 'Lychee']

for item in list_A:
    if item == 'Durian':
        print(True)

list_B.remove('Durian')
list_A.insert(4, 'KiwiFruit')

print(len(list_A))
print(len(list_B))

if len(list_A) > len(list_B):
    print("List A contains more elements")
else:
    print("List B contains more elements")

print(list_A.index('Avocado'))

try:
    print(list_B.index('Durian'))
except ValueError:
    print("-1")

list_B.extend(['Passion Fruit', 'Pomelo'])

print(list_A[2])

print(list_A)
print(list_B)